corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -36,6 +36,8 @@
 @register.simple_tag
 def render_form_xml(form):
     xml = form.get_xml()
+    if isinstance(xml, unicode):
+        xml.encode('utf-8', errors='replace')
```

Comment:
`errors='replace'` should not be necessary since it should be possible to convert any unicode character to UTF-8